tests/integration/test_manhattan_plot.py

```diff
@@ -0,0 +1,178 @@
+import pandas
+
+import dash
+import dash_bio
+import dash_html_components as html
+
+from common_features import nested_component_layout, \
+    nested_component_app_callback
+
+_data = pandas.read_csv(
+    'https://raw.githubusercontent.com/plotly/dash-bio-docs-files/master/' +
+    'manhattan_data.csv'
+)
+
+
+def test_dbmp001_genomewideline_value(dash_duo):
```

Comment:
this is a comment for all the tests here in general, I feel most of the tests share the same pattern, or the same common function, this is a sign for `pytest.mark.parametrize()`

Reply:
@byronz It appears to me as though the `pytest.mark.parametrize()` feature should be used when testing the same parameter with different values -- the tests in this PR test multiple parameters, but each with only one value. (This is obviously something that I'll be fixing in the future, and I think it would be appropriate to use this suggestion at that time. Please correct me if I'm wrong!)

Reply:
you can do it with multiple *parameters*, but that also means the result permutation is significantly big (it's a like a nested loop).  I just pointed out one possible alternative, but it's totally up to you to structure the tests :)